exchange/exchange.go

```diff
@@ -550,6 +555,33 @@ func (e *exchange) HoldAuction(ctx context.Context, r *AuctionRequest, debugLog
 	}, nil
 }
 
+// getBidderPreferredMediaTypeMap reads the preferred media type from the request and account and returns a map of bidder to preferred media type. Preference given to the request over account.
```

Comment:
`getBidderPreferredMediaTypeMap` sounds like the function is returning a map. Can we rename to simply `getBidderPreferredMediaType`?

Reply:
Done